src/hooks/useUserPermissions.ts

```diff
@@ -0,0 +1,19 @@
+import { useCurrentUser } from './useCurrentUser';
+
+export const useUserPermissions = () => {
+  const user = useCurrentUser();
+
+  const hasPermission = (permission: string | string[]) => {
+    return !user
```

Comment:
chỗ này em tách điều kiện !user thành 1 cái if cho dễ đọc code nhé, đang là toán tử 3 ngôi lồng nhau nữa nên khó đọc